dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js

```diff
@@ -586,7 +586,7 @@ export default {
   TargetDataBase: '目标库',
   TargetTable: '目标表',
   TargetJobName: 'TIS目标任务名',
-  'Please enter TIS DataX job name': '请输入TIS DataX任务名',
+  'Please enter PIGEON DataX job name': '请输入TIS DataX任务名',
```

Comment:
Maybe we should change the describe here, about the keyword `DataX` and `TIS`, BTW, please change `en_US.js` too

Reply:
> 'Please enter DataX job name': '请输入DataX任务名',

this shoud be OK

Reply:
> Maybe we should change the describe here, about the keyword `DataX` and `TIS`, BTW, please change `en_US.js` too

description has been improved, and en_US.js lack these description,  now, has added